app/controllers/orchestrations/migration_controller.rb

```diff
@@ -0,0 +1,36 @@
+# Allows to run migration orchestrations
+class Orchestrations::MigrationController < ApplicationController
+  def create
+    Orchestration.run kind: :migration
+    redirect_to root_path
+  end
+
+  def reboot_nodes
+    Minion.mark_pending_migration
+    ::Velum::Salt.update_orchestration_after_migration
+
+    redirect_to root_path
+  end
+
+  def check_mirror
+    Velum::Salt.call(
+      action:      "cmd.run",
+      arg:         "update-checker-migration",
+      targets:     "roles:(admin|kube-(master|minion))",
+      target_type: "grain_pcre"
+    )
```

Comment:
How long can this process take? Might it timeout the http request?

I wish we had something to background tasks properly :(

Reply:
this is really quick, i dont think it should make the request timeout